pkg/minishift/systemtray/icon/icon_notwin.go

```diff
@@ -0,0 +1,2408 @@
+	0x19, 0x2e, 0x1a, 0x87, 0x4b, 0xc6, 0xe1, 0xd8, 0xec, 0xc2, 0x67, 0xcf,
+	0xf9, 0x05, 0xed, 0x91, 0x09, 0xd8, 0x23, 0xc3, 0x49, 0xc0, 0x4b, 0x33,
+	0x8c, 0x46, 0xf7, 0xa8, 0xef, 0x36, 0x26, 0x78, 0x4c, 0x1b, 0xae, 0x8d,
+	0x0e, 0x91, 0xa4, 0x2d, 0x8d, 0xc3, 0x29, 0xc0, 0x19, 0xd1, 0x1d, 0x1a,
+	0xac, 0x04, 0x63, 0xc0, 0x19, 0x09, 0xce, 0x1e, 0x86, 0x1b, 0xa2, 0x7b,
+	0xea, 0xc0, 0x21, 0x79, 0x1a, 0xa6, 0x4e, 0xab, 0x38, 0x36, 0x17, 0x2f,
+	0x48, 0x4f, 0x07, 0x66, 0x46, 0x37, 0x69, 0x30, 0x12, 0x7c, 0xba, 0x0d,
+	0x27, 0x47, 0x77, 0x48, 0xd2, 0xbd, 0x2d, 0x85, 0x99, 0x73, 0xe0, 0x6a,
+	0x60, 0xaf, 0xe8, 0x16, 0x85, 0xe8, 0x02, 0x3f, 0x48, 0x70, 0x26, 0x70,
+	0x76, 0x1b, 0xee, 0x8a, 0x0e, 0xaa, 0x2a, 0x87, 0xe4, 0x6d, 0x94, 0xa1,
+	0x35, 0x01, 0x87, 0xe7, 0x62, 0x40, 0x3a, 0x11, 0x98, 0x17, 0xdd, 0xa4,
+	0xc1, 0x4b, 0x70, 0x48, 0xbb, 0xb8, 0x19, 0x53, 0x92, 0x4a, 0x67, 0x1c,
+	0xfe, 0x16, 0xcf, 0x6f, 0x6f, 0xbc, 0x04, 0x6b, 0x33, 0x7c, 0x19, 0x38,
+	0x6b, 0x02, 0x2e, 0x3c, 0x01, 0x26, 0xa3, 0x9b, 0xaa, 0xc4, 0x21, 0x79,
+	0x2b, 0x2d, 0x83, 0x03, 0x13, 0x9c, 0x98, 0x8a, 0xe1, 0x78, 0xbf, 0xe8,
+	0x1e, 0xc5, 0xc9, 0x70, 0xd1, 0x28, 0x3c, 0x35, 0xba, 0x43, 0x92, 0xee,
+	0xcf, 0x4a, 0x98, 0xb7, 0x01, 0xae, 0xcd, 0x30, 0x3f, 0xba, 0x45, 0xe5,
+	0x90, 0x8a, 0xa7, 0xc2, 0x9e, 0x9f, 0xe1, 0xf4, 0x11, 0xb8, 0x22, 0xba,
+	0xa7, 0x0a, 0x1c, 0x92, 0x1f, 0xc0, 0x32, 0xd8, 0x79, 0x08, 0x8e, 0x9f,
+	0xda, 0x4e, 0x71, 0x38, 0x7e, 0xbd, 0x04, 0x24, 0x78, 0x66, 0xdb, 0xbb,
+	0xc7, 0x25, 0x95, 0xdc, 0x38, 0x9c, 0x06, 0xbc, 0x31, 0xba, 0x43, 0xa5,
+	0x34, 0x01, 0x9c, 0x99, 0xe1, 0x53, 0xa3, 0x70, 0x53, 0x74, 0x4c, 0x59,
+	0x39, 0xf4, 0xdd, 0xcb, 0x4a, 0x98, 0xbd, 0xa1, 0xd8, 0x67, 0x7c, 0x32,
+	0xf0, 0x4c, 0xdc, 0x67, 0xac, 0x7b, 0x5a, 0xd1, 0x86, 0xe1, 0x04, 0x39,
+	0x3a, 0x44, 0x92, 0x1e, 0xc8, 0x04, 0xec, 0xdd, 0x85, 0x5f, 0xe0, 0x49,
+	0x56, 0xba, 0x7f, 0x1b, 0x81, 0xaf, 0x26, 0x38, 0x6b, 0x16, 0x5c, 0xb0,
+	0x00, 0xd6, 0x47, 0x07, 0x95, 0x89, 0x43, 0xf2, 0x94, 0x65, 0x70, 0xe8,
+	0x10, 0x9c, 0x32, 0x75, 0x42, 0xc5, 0xae, 0xd1, 0x3d, 0x2a, 0xa7, 0x04,
+	0xff, 0xbb, 0x0d, 0x1f, 0x88, 0xee, 0x90, 0xa4, 0xad, 0x31, 0x0e, 0x5f,
+	0x05, 0x9e, 0x11, 0xdd, 0xa1, 0xf2, 0x4b, 0xc5, 0x91, 0xa6, 0x5f, 0x4a,
+	0x70, 0xe6, 0x42, 0xb8, 0xd8, 0xc5, 0xa0, 0x86, 0x0f, 0xc9, 0x63, 0xb0,
+	0x57, 0x2a, 0x86, 0xe2, 0x97, 0x03, 0x0b, 0xa2, 0x7b, 0x54, 0x7a, 0x1b,
+	0x36, 0xc0, 0xa3, 0x0e, 0x81, 0x5b, 0xa2, 0x43, 0x24, 0x69, 0x6b, 0x8c,
+	0xc1, 0x9f, 0x26, 0x38, 0x3f, 0xba, 0x43, 0x95, 0x73, 0x2d, 0xf0, 0xd9,
+	0x16, 0x7c, 0x62, 0x18, 0x56, 0x46, 0xc7, 0x44, 0x69, 0xdc, 0x90, 0xbc,
+	0x14, 0xe6, 0xcf, 0x81, 0xe3, 0x32, 0x9c, 0x9c, 0xe0, 0x28, 0x1a, 0xf8,
+	0x35, 0xd0, 0x76, 0xfb, 0xec, 0x48, 0xf1, 0x43, 0x95, 0x24, 0x55, 0xc2,
+	0xb7, 0x61, 0xc6, 0x6e, 0xf0, 0x2b, 0xe0, 0x91, 0xd1, 0x2d, 0xaa, 0xa6,
+	0x04, 0x97, 0x75, 0x8b, 0xed, 0x18, 0x9f, 0x39, 0x00, 0x56, 0x45, 0xf7,
+	0x0c, 0x52, 0x23, 0x06, 0xc4, 0x73, 0x61
```

Comment:
@anjannath @gbraad Why can not we use gobindata for this? This does not make any sense from code point of view. For example if someone is going through code repository wont make any sense from code readability point of view.

Reply:
Personally I will prefer to have icons checked in to the code and then use those by gobindata.  cc @minishift/minishift-dev  WDYT?

Reply:
these are resources that do not really have to exist as a 'file'. I also include certs in the source code... is this really a problem? It is not something we need to maintain. It is an include and forget... it is even in a separate file. I see no problem with this.

Reply:
@LalatenduMohanty @gbraad The byte arrays of the images are generated using https://github.com/cratonica/2goarray, if we don't want to check-in these files in the tree, we could generate them on the go (with a little bit of hack) like we do for the default add-ons. That way we have to check-in the actual image files in tree

But add-ons keep changing, which requires them to be regenerated, but these icons won't change that much i suppose.

Reply:
> these icons won't change that much i suppose.

same thought here. If they would have to change, it would likely be once. it would not justify the additional hassle with bindata. ISO's to include in the build process is a different thing. (Also, Minishift is currently free of using bindata(?) for actual inclusion... and i would almost urge to get rid of this, and only add this for use with CDK).

Reply:
@gbraad We still use go-bindata for packing the default add-ons to the binary.

We are not going to change the icons that often, we decided about an icon for minishift after a long time and the status icons (red-> sopped, green->running, grey->does not exist) are also very clear, so they are in effect static resources.

Also, the files are under the `icon` package `pkg/minishift/systemtray/icon/*` that makes it clear that these are the icon data.

But this does take away the ability to easily make changes to the icons, for that we can add an script, and put the image files in tree. But since those once added will not change that often, there is no point in auto-generating them while building.

Reply:
> But this does take away the ability to easily make changes to the icons, for that we can add an script, and put the image files in tree. But since those once added will not change that often, there is no point in auto-generating them while building.
> cc @minishift/minishift-dev

This will be a better solution than the current one as the maintainability of code will be better for future.